libtsuba/src/RDG.cpp

```diff
@@ -1026,6 +1031,37 @@ katana::RDG::SetEdgeEntityTypeIDArrayFile(
   return core_->RegisterEdgeEntityTypeIDArrayFile(new_type_id_array.BaseName());
 }
 
+katana::Result<katana::RDKLSHIndex>
+katana::RDG::LoadRDKLSHIndex() {
+  if (!KATANA_EXPERIMENTAL_ENABLED(UnstableRDGStorageFormat)) {
+    return KATANA_ERROR(
+        ErrorCode::InvalidArgument,
+        "The UnstableRDGStorageFormat feature flag must be set to use this "
+        "feature");
+  }
+  std::string path =
+      KATANA_CHECKED(core_->part_header().OptionalDatastructureManifest(
+          kOptionalDatastructureRDKLSHIndex));
+  katana::RDKLSHIndex index =
+      KATANA_CHECKED(katana::RDKLSHIndex::Load(rdg_dir(), path));
```

Comment:
We should return an appropriate error if this fails

Reply:
the rdg api now returns the following error on failure:
```
  katana::RDKLSHIndex index = KATANA_CHECKED_CONTEXT(
      katana::RDKLSHIndex::Load(rdg_dir(), path),
      "Failed to load RDKLSHIndex located at {}", path);
```